src/ol_infrastructure/applications/open_metadata/__main__.py

```diff
@@ -0,0 +1,180 @@
+            from_port=DEFAULT_POSTGRES_PORT,
+            to_port=DEFAULT_POSTGRES_PORT,
+            description="Access to postgres from open metadata servers.",
+        ),
+        ec2.SecurityGroupIngressArgs(
+            security_groups=[],
+            protocol="tcp",
+            from_port=DEFAULT_POSTGRES_PORT,
+            to_port=DEFAULT_POSTGRES_PORT,
+            cidr_blocks=["73.218.126.92/32", "98.110.175.231/32"],
+            description="Allow mike and matt to connect to postgres.",
+        ),
+    ],
+    vpc_id=apps_vpc["id"],
+    tags=aws_config.tags,
+)
+
+rds_defaults = defaults(stack_info)["rds"]
+rds_defaults["instance_size"] = (
+    open_metadata_config.get("db_instance_size") or DBInstanceTypes.small.value
+)
+
+open_metadata_db_config = OLPostgresDBConfig(
+    instance_name=f"open-metadata-db-{stack_info.env_suffix}",
+    password=open_metadata_config.get("db_password"),
+    subnet_group_name=apps_vpc["rds_subnet"],
+    security_groups=[open_metadata_database_security_group],
+    storage=open_metadata_config.get("db_capacity")
+    or str(AWS_RDS_DEFAULT_DATABASE_CAPACITY),
+    engine_major_version="15",
+    tags=aws_config.tags,
+    db_name="open_metadata",
+    **defaults(stack_info)["rds"],
+)
+open_metadata_db = OLAmazonDB(open_metadata_db_config)
+
+open_metadata_db_vault_backend_config = OLVaultPostgresDatabaseConfig(
+    db_name=open_metadata_db_config.db_name,
+    mount_point=f"{open_metadata_db_config.engine}-open-metadata",
+    db_admin_username=open_metadata_db_config.username,
+    db_admin_password=open_metadata_config.get("db_password"),
+    db_host=open_metadata_db.db_instance.address,
+)
+open_metadata_db_vault_backend = OLVaultDatabaseBackend(
+    open_metadata_db_vault_backend_config
+)
+
+open_metadata_db_consul_node = Node(
+    "open-metadata-postgres-db",
+    name="open-metadata-postgres-db",
+    address=open_metadata_db.db_instance.address,
+    opts=consul_provider,
+)
+
+open_metadata_db_consul_service = Service(
+    "open-metadata-instance-db-service",
+    node=open_metadata_db_consul_node.name,
+    name="open-metadata-db",
+    port=open_metadata_db_config.port,
+    meta={
+        "external-node": True,
+        "external-probe": True,
+    },
+    checks=[
+        ServiceCheckArgs(
+            check_id="open-metadata-instance-db",
+            interval="10s",
+            name="open-metadata-instance-id",
+            timeout="60s",
+            status="passing",
+            tcp=open_metadata_db.db_instance.address.apply(
+                lambda address: f"{address}:{open_metadata_db_config.port}"
+            ),
+        )
+    ],
+    opts=consul_provider,
+)
+
+# Create a vault policy to allow dagster to get to the secrets it needs
+open_metadata_server_vault_policy = vault.Policy(
+    "open-metadata-server-vault-policy",
+    name="open-metadata-server",
+    policy=Path(__file__)
+    .parent.joinpath("open_metadata_server_policy.hcl")
```

Comment:
This file doesn't exist.

But also we don't need _this_ specifically but we will need something like it to give the app / vault-secrets-operator the permissions it needs to generate db creds.

Reply:
I just forgot to add the file which I'd written locally. I put it in, even if we need something like it we can alter the contents once we figure them out :)